continuous-pipe.yml

```diff
@@ -1,323 +1,394 @@
-            - name: INSTALL_COMMON_PACKAGES
-              value: "false"
+            NODE_VERSION: "8.x"
+            INSTALL_COMMON_PACKAGES: "false"
         phantomjs2:
           image: quay.io/continuouspipe/phantomjs2
-          tag: latest
+          tag: ${FROM_TAG}
           reuse: false
         piwik_php71_apache:
           image: quay.io/continuouspipe/piwik-php7.1-apache
-          tag: latest
+          tag: ${FROM_TAG}
           reuse: false
+          environment:
+            PHP_VERSION: 7.1
         postgres94:
           image: quay.io/continuouspipe/postgres9.4
-          tag: latest
+          tag: ${FROM_TAG}
           reuse: false
+          environment:
+            FROM_TAG: '9.4'
         postgres96:
           image: quay.io/continuouspipe/postgres9.6
-          tag: latest
+          tag: ${FROM_TAG}
           reuse: false
+          environment:
+            FROM_TAG: '9.6'
         redis:
           image: quay.io/continuouspipe/redis3
-          tag: latest
+          tag: ${FROM_TAG}
           reuse: false
         redis_highly_available:
           image: quay.io/continuouspipe/redis3-highly-available
-          tag: latest
+          tag: ${FROM_TAG}
           reuse: false
         scala_sbt:
           image: quay.io/continuouspipe/scala-base
-          tag: latest
+          tag: ${FROM_TAG}
           reuse: false
         ssh_forward:
           image: quay.io/continuouspipe/ssh-forward
-          tag: latest
+          tag: ${FROM_TAG}
           reuse: false
         spryker_php71_nginx:
           image: quay.io/continuouspipe/spryker-php7.1-nginx
-          tag: latest
+          tag: ${FROM_TAG}
           reuse: false
+          environment:
+            PHP_VERSION: '7.1'
+            WEB_SERVER: nginx
         spryker_php71_apache:
           image: quay.io/continuouspipe/spryker-php7.1-apache
-          tag: latest
+          tag: ${FROM_TAG}
           reuse: false
+          environment:
+            PHP_VERSION: '7.1'
+            WEB_SERVER: apache
         symfony_php70_nginx:
           image: quay.io/continuouspipe/symfony-php7-nginx
-          tag: latest
+          tag: ${FROM_TAG}
           reuse: false
+          environment:
+            PHP_VERSION: '7'
+            WEB_SERVER: nginx
         symfony_php56_nginx:
           image: quay.io/continuouspipe/symfony-php5.6-nginx
-          tag: latest
+          tag: ${FROM_TAG}
           reuse: false
+          environment:
+            PHP_VERSION: '5.6'
+            WEB_SERVER: nginx
         symfony_php56_apache:
           image: quay.io/continuouspipe/symfony-php5.6-apache
-          tag: latest
+          tag: ${FROM_TAG}
           reuse: false
+          environment:
+            PHP_VERSION: '5.6'
+            WEB_SERVER: apache
         tideways:
           image: quay.io/continuouspipe/tideways
-          tag: latest
+          tag: ${FROM_TAG}
           reuse: false
 
 filter: code_reference.branch in ["master"]
```

Comment:
I re-added this until we could update the all the images - shall we give it a test on this branch by removing this filter?